docs/source/getstarted.rst

```diff
@@ -103,31 +108,39 @@ Next Steps - "Night-Night"
-    
+
+Now, nightnight.py is saved in the same folder as 01_hello_world.py.
+	
 2. Open this new document in a source code editor or plain-text editor. Free source code editors, such as `PyCharm Community Edition <https://www.jetbrains.com/pycharm/>`_ , `Atom <https://atom.io>`_ , `Sublime <https://www.sublimetext.com>`_ , or `TextWrangler <http://www.barebones.com/products/textwrangler/>`_ can be found online. Anki does not provide tech support for third-party source code editors.
 
-3. Each line in the program relates to a specific function. To learn more, see :doc:`the Beginner's Tutorial </tutorial-beginner>`.
+3. Each line in the program relates to a specific function.
 
-4. Move to the line that starts with "robot.say_text"
+    a. ``import cozmo`` allows your program to access the Cozmo SDK code contained within the ``cozmo`` module.
+    b. Text sandwiched between three ``'`` marks is a Docstring. Docstrings are like comments, and are placed inside code to give information to the user.
+    c. ``robot.say_text("Hello World").wait_for_completed`` is the core of the program.
+        i. ``robot.say_text(…)`` is the function that makes Cozmo speak a string out loud.
+        ii. ``"Hello World"`` is the string which Cozmo will speak.
+        iii. ``wait_for_completed()`` tells Cozmo to finish speaking before moving to the next line of code. Without this, our program would end before Cozmo said anything!
 
-  1. Select the phrase "Hello World". Do NOT select the parentheses or quotation marks around the phrase; those are necessary for Python to properly parse the command.
-  2. Type in the new phrase you would like Cozmo to say. In this example, Cozmo will say "Night Night"::
+4. Move to the line that starts with "robot.say_text"
 
-      robot.say_text("Night Night").wait_for_completed()
+    a. Select the phrase "Hello World". Do NOT select the parentheses or quotation marks around the phrase; those are necessary for Python to properly parse the command.
+    b. Type in the new phrase you would like Cozmo to say. In this example, Cozmo will say "Night Night”, so that line should look like this: ``robot.say_text("Night Night").wait_for_completed()``
 
 5. Save the nightnight.py file.
 6. Now you can run your program:
 
-        a. For macOS and Linux systems, type the following into the same window and press **Enter**::
+        a. For macOS and Linux systems, type the following into the same Terminal window and press **Enter**::
 
             ./nightnight.py
 
-        b. For Windows systems, type the following into the same window and press **Enter**::
+        b. For Windows systems, type the following into the same Command Prompt window and press **Enter**::
 
             py nightnight.py
 
 7. If done correctly, Cozmo will say the new phrase.
 
-----
+
+Now that you have written your own Cozmo program, take a look at the rest of the Cozmo SDK package, and at the many other example programs to get more ideas.
```

Comment:
maybe get rid of the word package there entirely? - e.g. "take a look at the rest of the Cozmo SDK and at the many other example programs to get more ideas."?